R/draws.R

```diff
@@ -100,7 +104,13 @@
 #' (performed with the function ([impute()]). To summarize, **at this stage only pre-ICE data
 #' and post-ICE data that is after ICEs for which MAR imputation is specified are used**.
 #'
-#' The vars argument is a named list that specifies the names of key variables within
+#' If the `data_ice` argument is omitted, or if a subject doesn't have a record within it, then it is
+#' assumed that a hypothetical ICE occurred after the last visit for all impacted subjects and as such all missing data
+#' will be imputed under MAR. Please note that the ICE visit cannot be changed by the `update_strategy` argument
+#' in [impute()] and as such the subjects missing data will still be imputed under MAR even if you change the
+#' subjects imputation strategy.
```

Comment:
Is this comment needed? E.g.  when changing from MAR to non-MAR we should throw an error.

Reply:
It won't throw an error as there is no Post-ICE data

Reply:
ok, so in this case if the user changes the imputation strategy to "JR" in `impute()`, the function will impute again under MAR? It makes sense to me, since the ICE did not occurr and so setting JR is just meaningless. Should we throw a warning, e.g. "you are setting a JR strategy for a patient that did not experienced any ICE. The imputation will be performed under MAR" (also for CRAN release if now there is no time).

Reply:
My gut feeling is that would be overkill. At least we have it clear in the documentation that not setting the ice visit locks it at visit + 1  and also that pre-ice visits are always imputed with MAR.

I'm also feeling that if someone was being lazy and just did a blanket "impute everything with JR" it would be annoying to then have to go through and individually remove all the patients that didn't have an ICE.

Reply:
agreed!